### ISSUE TITLE
Local model is not working

### ISSUE THREAD

**Issue by bartoszpankratz** [tolcz/ABCDeGraphGenerator.jl#3]
Hi,

So after the tests of the newest version of the code (the one with a fix) it turns out that for the local model code is throwing errors for entire sweep of ξ parameter. E.g, for graph with 100000 nodes, ξ equal to 0.2, β 1.1,  γ 2.5 and min_deg  equal to 5 it looks like that:

```Julia
Warning: ((ArgumentError("μ is too large to generate a graph"), Base.StackTraces.StackFrame[macro expansion at logging.jl:340 [inlined], (::var"#3#6"{ArgumentError})() at get_ABCDe.jl:61, with_logstate(f::Function, logstate::Any) at logging.jl:491, with_logger(f::Function, logger::Base.CoreLogging.SimpleLogger) at logging.jl:603, top-level scope at get_ABCDe.jl:60, eval at boot.jl:360 [inlined], include_string(mapexpr::typeof(identity), mod::Module, code::String, filename::String) at loading.jl:1094, _include(mapexpr::Function, mod::Module, _path::String) at loading.jl:1148, include(mod::Module, _path::String) at Base.jl:386, exec_options(opts::Base.JLOptions) at client.jl:285, _start() at client.jl:485]), "  for params: 100000 0.2 1.1 2.5 5 true")
└ @ Main /root/get_ABCDe.jl:61

```
By the fact that it behave like that for all ξs and is not replicable with og ABCD package I believe that it is only some minor typo.

All the best

**Comment by tlcz** [comment]
Hi
I am confused which parameter you have in mind ξ or  μ? It is illegal to specify ξ for a local model.
https://github.com/tolcz/ABCDeGraphGenerator.jl/blob/7a76d830c6970ad82e2d93c86429f4338f24c765/src/graph_sampler.jl#L33-L35
If you still experience the error could you pls attach a sample configuration to reproduce. Thank you.

**Comment by tolcz** [comment]
It is moreover unclear from the error message which function throws it - `CL_model()` or `config_model()`? Please specify.

**Comment by tlcz** [comment]
Hi @bartoszpankratz 
The bug was introduced by 5cd4f42fe7fc90d23dae9b2874879f6aa87872f9. Fixed with 34e502a96e76525b978d5706146fdea130019f1b. Thank you!
